pkg/backup/item_backupper.go

```diff
@@ -188,7 +189,8 @@ func (ib *itemBackupper) backupItem(logger logrus.FieldLogger, obj runtime.Unstr
 	name = metadata.GetName()
 	namespace = metadata.GetNamespace()
 
-	if groupResource == kuberesource.PersistentVolumes {
+	// #4758 Do not take snapshot for PV to avoid duplicated snapshotting, when CSI feature is enabled.
+	if groupResource == kuberesource.PersistentVolumes && !features.IsEnabled(velerov1api.CSIFeatureFlag) {
```

Comment:
After making CSI plugin GA, the `CSIEnable` feature flag is still there?
And even though the flag is still there, it is just a flag, if the CSI plugin isn't installed, no one will take snapshot with the changes